erpnext/accounts/doctype/bank_transaction/bank_transaction.py

```diff
@@ -307,6 +311,26 @@ def auto_set_party(self):
 
 		self.party_type, self.party = result
 
+	def handle_included_fee(self):
+		"""
+		The included_fee is only handled for withdrawals. An included_fee for a deposit, is not credited to the account and is
+		therefore outside of the deposit value and can be larger than the deposit itself.
+		"""
+
+		if self.included_fee and self.withdrawal:
+			if self.included_fee > self.withdrawal:
+				frappe.throw(_("Included fee is bigger than the withdrawal itself."))
```

Comment:
This should likely be called `validate_included_fee` and get called on `validate`, not `before_validate`.